packages/java/hilla-dev-mode/src/main/resources/META-INF/resources/frontend/dev-tools-database.ts

```diff
@@ -16,12 +16,12 @@ type H2Data = {
 };
 @customElement('devtools-database')
```

Comment:
I guess, we should get rid of decorators completely here

Reply:
Should we? Class level decorators have no issues as far as I know

Reply:
Yes but it looks strange if you use class decorators but not the property ones. And we could also avoid any possible issues with the decorators at all

Reply:
Ok, now there are no decorators